src/client/rekor.ts

```diff
@@ -52,11 +50,8 @@ export class Rekor {
         'User-Agent': ua.getUserAgent(),
       },
     });
-    this.baseUrl = Rekor.getBaseUrl(options.baseURL);
-  }
 
-  public static getBaseUrl(baseURL?: string) {
-    return baseURL ?? DEFAULT_BASE_URL;
+    this.baseUrl = options.baseURL;
```

Comment:
Moved the default URL higher in the stack so that we have fewer optional parameters to juggle.